OpenRA.Mods.Common/ServerTraits/SkirmishLogic.cs

```diff
@@ -98,6 +98,14 @@ static bool TryInitializeFromFile(S server, string path, Connection conn)
 				var client = server.GetClient(conn);
 				SkirmishSlot.DeserializeToClient(playerNode.Value, client);
 				client.Color = LobbyCommands.SanitizePlayerColor(server, client.Color, client.Index);
+
+				var selectableFactions = server.Map.WorldActorInfo.TraitInfos<FactionInfo>()
+					.Where(f => f.Selectable)
+					.Select(f => f.InternalName)
+					.ToList();
```

Comment:
`ToList` not strictly required.

Reply:
it would be more eficient to not use select as well, but just use a Any() call on the trait infos